test/unit/org/apache/cassandra/index/sai/utils/SAIRandomizedTester.java

```diff
@@ -129,6 +129,16 @@ public static int between(int min, int max)
         return getRandom().nextIntBetween(min, max - 1);
     }
 
+    public static double nextDouble()
+    {
+        return randomDoubleBetween(0, 1);
```

Comment:
nit: I think this could also just be `getRandom().nextDouble()`?

Reply:
`randomDoubleBetween` calls `getRandom().nextDouble()` under the hood. It seems more clear to me to use the `randomDoubleBetween` with the max and mix explicitly stated. Though, now as you mentioned, I noticed that `nextBoolean` uses directly `getRandom().nextBoolean()`.  I don't have strong preference for one or the other. I can change it

Reply:
Yeah, it's equivalent...just seemed less complicated to go directly to `getRandom().nextDouble()`

Reply:
Done